deploy/deploy-scaledobject.yaml

```diff
@@ -12,7 +12,7 @@ spec:
   triggers:
```

Comment:
Do we still need this file when our Helm chart is completed?

Reply:
Or is  this to attach a raw YAML file in a release similar to KEDA core?

Reply:
The `ScaledObject` is not part of Helm chart. Users can use the file as template for creating their own `ScaledObject`s.

Reply:
Deleted `deploy/deploy.yaml` as it is no longer needed.

Reply:
Thank you! And no problem related to the SO, I clearly was not paying attention!